src/app/starredentries/starredentries.component.html

```diff
@@ -16,110 +16,248 @@
+                    <div *ngIf="tool?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center"></div>
+                  </mat-card-header>
+                </div>
+                <app-starring class="pull-right" [tool]="tool" (starGazersChange)="starGazersChange()"></app-starring>
+              </div>
+              <div>
+                <hr />
+                <div fxLayout="row" fxLayoutAlign="space-between stretch">
+                  <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                    <small class="spacing">Last updated {{ tool?.lastUpdated | date }}</small>
+
+                    <mat-chip-list>
+                      <mat-chip class="container-label-tag" *ngFor="let label of tool?.labels | slice: 0:3">{{ label.value }}</mat-chip>
+                      <mat-chip class="gray-label-tag" *ngFor="let descriptortype of tool?.descriptorType">
+                        {{ descriptortype }}
+                      </mat-chip>
+                    </mat-chip-list>
 
-        <mat-list>
-          <mat-list-item *ngFor="let workflow of starredWorkflows; let last = last">
-            <h3 class="spacing" mat-line>
-              <a routerLink="/workflows/{{ workflow.full_workflow_path }}" target="_blank" rel="noopener">
-                {{ workflow?.organization + '/' + workflow?.repository + (workflow?.toolname ? '/' + workflow?.toolname : '') }}
-              </a>
-            </h3>
-            <span mat-line>
-              <small class="spacing">Last updated {{ workflow?.lastUpdated | date }}</small>
-              <span class="spacing">{{ workflow?.descriptorType | uppercase }}</span>
-              <a class="spacing" *ngIf="workflow?.provider" [href]="workflow?.providerUrl">{{ workflow?.provider }}</a>
-            </span>
-            <app-starring
-              class="pull-right"
-              style="width: 100px"
-              [workflow]="workflow"
-              (starGazersChange)="starGazersChange()"
-            ></app-starring>
-            <mat-divider></mat-divider>
-          </mat-list-item>
-        </mat-list>
+                    <span id="verifiedIcon" *ngIf="(tool$ | async)?.versionVerified" fxHide.lt-sm>
+                      <mat-chip matTooltip="Verified" class="gray-label-tag">
+                        <a [href]="verifiedLink" class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
+                          <mat-icon class="verified-icon">done</mat-icon>
+                          <span>Verified</span>
+                        </a>
+                      </mat-chip>
+                    </span>
+                  </div>
+                  <div><button mat-raised-button color="accent" class="view-button">View</button></div>
+                </div>
+              </div>
+            </a.no-underline>
+          </mat-card>
+        </div>
       </div>
     </div>
   </mat-tab>
 
-  <mat-tab label="Organizations">
+  <!-- Services not done -->
```

Comment:
If they're not done, should you remove the tab until they are?

I see now that my comment about your screenshot may only be applicable to this case, which may be n/a.